civis/cli/__main__.py

```diff
@@ -130,11 +136,12 @@ def param_case_map(param_names):
     return result
 
 
-def invoke(method, path, op, *args, json_output=False, **kwargs):
+def invoke(method, path, op, *args, **kwargs):
 
     # Remove None b/c click passes everything in as None if it's not set.
     kwargs = {k: v for k, v in kwargs.items()
               if v is not None}
+    json_output = kwargs.setdefault('json_output', False)
```

Comment:
The code in this function iterates over `kwargs`, and that never used to contain `json_output`. This line should be something like `json_output = kwargs.pop('json_output', False)` to keep it working the same as it did before.